Mathlib/Topology/Instances/AddCircle.lean

```diff
@@ -137,6 +137,9 @@ theorem coe_sub (x y : 𝕜) : (↑(x - y) : AddCircle p) = (x : AddCircle p) -
 theorem coe_neg {x : 𝕜} : (↑(-x) : AddCircle p) = -(x : AddCircle p) :=
   rfl
 
+theorem coe_zero : ↑(0 : 𝕜) = (0 : AddCircle p) :=
```

Comment:
```suggestion
@[norm_cast]
theorem coe_zero : ↑(0 : 𝕜) = (0 : AddCircle p) :=
```

Reply:
I guess this applies to all these lemmas.

Reply:
I'll add this attr to this PR, then open a secondary PR to `norm_cast` the other lemmas :+1:

Reply:
Fine to leave the attribute till the next PR too.